src/main/java/org/apache/flink/benchmark/TwoInputBenchmark.java

```diff
@@ -67,4 +68,23 @@ public void twoInputMapSink(FlinkEnvironmentContext context) throws Exception {
 
 		env.execute();
 	}
+
+	@Benchmark
+	public void twoInputOneIdleMapSink(FlinkEnvironmentContext context) throws Exception {
+
+		StreamExecutionEnvironment env = context.env;
+		env.enableCheckpointing(CHECKPOINT_INTERVAL_MS);
+		env.setParallelism(1);
+
+		QueuingLongSource.reset();
+		DataStreamSource<Long> source1 = env.addSource(new QueuingLongSource(1, RECORDS_PER_INVOCATION - 1));
```

Comment:
One last thing. I've checked the benchmark results and unfortunately this one is very slow with 25M of records. However it is I think also more stable then the non idle one, so I think we should be fine with changing number of records to `15M` for `twoInputOneIdleMapSink`. 

I think that we can put `@OperationsPerInvocation` annotation at the method level to have two benchmarks in the same file running with different number of records.

Reply:
Yes, I'll revise it right away.

Reply:
The code has been updated. @pnowojski